packages/core/src/__tests__/index.ts

```diff
@@ -360,13 +360,13 @@ describe('@reatom/core', () => {
       expect(store.getState(count)).toBe(4)
       expect(store.getState().countDoubled).toBe(8)
     })
-    test('createStore with nullable atom', () => {
+    test('createStore with undefined atom', () => {
       const increment = declareAction()
       const countStatic = declareAtom(['countStatic'], 0, handle =>
         handle(increment, state => state + 1),
       )
 
-      const store = createStore(null, { countStatic: 10 })
+      const store = createStore(undefined, { countStatic: 10 })
```

Comment:
:confused: why not `null`? the code looks strange in this case

Reply:
Function with null, cant have default value

Reply:
It's not like the usual code in all libraries.

Reply:
You should not use null as undefined in javascript, null is value, but we used it as no value provided, we can't use default argument value for that